db/schema.rb

```diff
@@ -56,6 +56,15 @@
     t.index ["pet_id"], name: "index_adopter_applications_on_pet_id"
   end
 
+  create_table "adopter_foster_accounts", force: :cascade do |t|
```

Comment:
This table should not be committed as we deleted it recently. Please remove this (might want to rebase with main branch)